python/ray/worker.py

```diff
@@ -1682,16 +1682,29 @@ def kill(actor):
 
 
 def cancel(object_id, force=False):
-    """Kill a task forcefully.
+    """Cancels a locally-submitted task according to the following pseudocode.
 
-    This will interrupt any running tasks on the actor, causing them to fail
-    immediately. Any atexit handlers installed in the actor will still be run.
+    Task is PENDING execution:
+        Pending task is canceled.
```

Comment:
Saying "pending task is canceled" doesn't add much on top of the function name. Maybe say "if the task is pending, it will not be executed."